arviz/data/io_cmdstan.py

```diff
@@ -0,0 +1,417 @@
+"""CmdStan-specific conversion code."""
+from collections import defaultdict
+from copy import deepcopy
+from glob import glob
+import linecache
+import os
+import re
+
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from .inference_data import InferenceData
+from .base import requires, dict_to_dataset, generate_dims_coords
+
+
+class CmdStanConverter:
+    """Encapsulate CmdStan specific logic."""
+
+    # pylint: disable=too-many-instance-attributes
+
+    def __init__(self, *, output=None, prior=None, posterior_predictive=None,
+                 observed_data=None, observed_data_var=None,
+                 log_likelihood=None, coords=None, dims=None):
+        self.output = glob(output) if isinstance(output, str) else output
```

Comment:
I don't complete understand what glob is doing or what the expected input for output is. Not saying its not just unsure at the moment

Reply:
Looks like `output` is either a glob pattern (`output/*.csv`) or a list of strings, and similar with `prior`